docs/USAGE.md

```diff
@@ -1,11 +1,25 @@
 # Instructions to run the application locally
- 
- *If you're running a [`v1.x` release JAR file](https://github.com/MobilityData/gtfs-validator/releases) you'll need Java 11, and can follow [these instructions](https://github.com/MobilityData/gtfs-validator/tree/v1.4.0#via-java-on-your-local-computer). The below instructions are for the master branch, which will be v2.0.*
+*If you're running a [`v1.x` release JAR file](https://github.com/MobilityData/gtfs-validator/releases) you'll need Java 11, and can follow [these instructions](https://github.com/MobilityData/gtfs-validator/tree/v1.4.0#via-java-on-your-local-computer). The below instructions are for the master branch, which will be v2.0.*
 
-1. Install [Java 8 or higher](https://www.oracle.com/java/technologies/javase-downloads.html)
-1. Download the latest gtfs-validator JAR file from our [Releases page](https://github.com/MobilityData/gtfs-validator/releases) or snapshot artifact from [GitHub Actions](https://github.com/MobilityData/gtfs-validator/actions?query=branch%3Amaster) or [Circle-CI Pipelines](https://app.circleci.com/pipelines/github/MobilityData/gtfs-validator?branch=master)
+1. Install [Java 8 or higher](https://www.oracle.com/java/technologies/javase-downloads.html).
+1. Download the latest gtfs-validator JAR file from our [Releases page](https://github.com/MobilityData/gtfs-validator/releases) or snapshot artifact from [GitHub Actions](https://github.com/MobilityData/gtfs-validator/actions?query=branch%3Amaster).
 
 ## via cli-app
+**Full list of command line parameters available**
+
+| Short name 	| Long name             	| required? 	| Description                                                                                                               	|
+|------------	|-----------------------	|-----------	|---------------------------------------------------------------------------------------------------------------------------	|
+| `-i`       	| `--input`             	| `false`   	| Location of the input GTFS ZIP or unarchived directory.                                                                   	|
```

Comment:
Instead of "true" and "false" for required and optional, I would put in the table for "Required" and "Optional". Header can stay the same but should be capitalized.